config.rb

```diff
@@ -72,8 +72,10 @@
 end
 
 for url in %w[d/random-with-units documentation/breaking-changes/random-with-units
-    d/breaking-changes/color-units documentation/breaking-changes/color-units] do
-  redirect url, to: "/documentation/breaking-changes/function-units"
+    d/color-units documentation/breaking-changes/color-units] do
+  # Middleman requires the origin URL to contain an extension, otherwise it will
```

Comment:
I think it's GitHub pages' behavior that's causing this, since we don't run Middleman as a server in production.